scripts/bootstrap.sh.in

```diff
@@ -209,86 +147,28 @@ sync_salt() {
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.apiserver-kubelet-client saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.sa saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.etcd-ca saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.etcd-healthcheck-client saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.etcd-peer saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.etcd-server saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.apiserver-etcd-client saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.front-proxy-ca saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.certs.front-proxy-client saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-}
-
-run_kubeconfig() {
-    "$SALT_CALL" --retcode-passthrough state.apply \
-        metalk8s.kubeadm.init.kubeconfig saltenv=metalk8s-@VERSION@ \
-        pillarenv=metalk8s-@VERSION@
-}
-
-run_control_plane() {
-    "$SALT_CALL" --retcode-passthrough state.apply \
-        metalk8s.kubeadm.init.control-plane saltenv=metalk8s-@VERSION@ \
-        pillarenv=metalk8s-@VERSION@
-}
-
-run_etcd() {
-    "$SALT_CALL" --retcode-passthrough state.apply \
-        metalk8s.kubeadm.init.etcd saltenv=metalk8s-@VERSION@ \
-        pillarenv=metalk8s-@VERSION@
-}
-
-run_mark_control_plane() {
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.kubeadm.init.mark-control-plane saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
-}
-
-install_addons() {
-    "$SALT_CALL" --retcode-passthrough state.apply \
-        metalk8s.kubeadm.init.addons saltenv=metalk8s-@VERSION@ \
-        pillarenv=metalk8s-@VERSION@
+sync_local() {
+    ${SALT_CALL} --local --retcode-passthrough saltutil.sync_all saltenv=metalk8s-@VERSION@
 }
 
-install_calico() {
-    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.calico saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
+orchestrate_bootstrap() {
+    ${SALT_CALL} --local --retcode-passthrough state.orchestrate metalk8s.orchestrate.bootstrap_step1 saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
+    set_salt_command
+    sync_salt
+    ${SALT_MASTER_CALL}-run state.orchestrate metalk8s.orchestrate.bootstrap_step2 saltenv=metalk8s-2.0
```

Comment:
Not sure but I'd say:
```suggestion
    ${SALT_MASTER_CALL}-run state.orchestrate metalk8s.orchestrate.bootstrap_step2 saltenv=metalk8s-@VERSION@
```

And we don't need `pillarenv=metalk8s-@VERSION@` in here?